app/assets/stylesheets/control.scss

```diff
@@ -0,0 +1,8 @@
+.control {
+  .file {
+    .for {
+      .violations {
```

Comment:
Empty rule<br>Selector should have depth of applicability no greater than 2, but was 4